plan/statistics/statistics.go

```diff
@@ -336,36 +575,38 @@ func TableFromPB(ti *model.TableInfo, tpb *TablePB) (*Table, error) {
 	if len(tpb.Columns) != len(ti.Columns) {
 		return nil, errors.Errorf("column count not match, expected %d, got %d", len(ti.Columns), len(tpb.Columns))
 	}
+	if len(tpb.Indices) != len(ti.Indices) {
```

Comment:
If the indices counts don't match, it may means user create an new index which have not been analysised, you should handle this case.